Jenkinsfile

```diff
@@ -8,7 +8,7 @@ pipeline {
         // It is recommended by SQUARE team do not add the label and let the
         // sytem decide.
         docker {
-            image 'lsstts/aos:w_2020_14'
+            image 'lsstts/aos:w_2020_15'
```

Comment:
Should this work? It's much better to define just once.. You can still change SIMS_VERSION independent of the docker version if needed.

```patch
diff --git a/Jenkinsfile b/Jenkinsfile
index 060f30b..f603c59 100644
--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -2,13 +2,26 @@

 pipeline {

+    environment {
+        // Development tool set
+        DEV_TOOL="/opt/rh/devtoolset-8/enable"
+        // Position of LSST stack directory
+        LSST_STACK="/opt/lsst/software/stack"
+        // Pipeline Sims Version
+        SIMS_VERSION="sims_w_2020_15"
+        // XML report path
+        XML_REPORT="jenkinsReport/report.xml"
+        // Module name used in the pytest coverage analysis
+        MODULE_NAME="lsst.ts.ofc"
+    }
+
     agent {
         // Use the docker to assign the Python version.
         // Use the label to assign the node to run the test.
         // It is recommended by SQUARE team do not add the label and let the
         // sytem decide.
         docker {
-            image 'lsstts/aos:w_2020_15'
+            image "lsstts/${env.SIMS_VERSION}"
             args '-u root'
         }
     }
@@ -17,18 +30,6 @@ pipeline {
         pollSCM('H * * * *')
     }

-    environment {
-        // Development tool set
-        DEV_TOOL="/opt/rh/devtoolset-8/enable"
-        // Position of LSST stack directory
-        LSST_STACK="/opt/lsst/software/stack"
-        // Pipeline Sims Version
-        SIMS_VERSION="sims_w_2020_15"
-        // XML report path
-        XML_REPORT="jenkinsReport/report.xml"
-        // Module name used in the pytest coverage analysis
-        MODULE_NAME="lsst.ts.ofc"
-    }

     stages {
         stage ('Install Requirements') {
```

Reply:
This method will give the error:
```
/home/jenkins/workspace/_Site_ts_ofc_feature_testJenkins@tmp/durable-94fae833/script.sh: line 1: lsstts/aos:${env.LSST_VERSION}: bad substitution
```
and

```
Error when executing always post condition:
org.jenkinsci.plugins.workflow.steps.MissingContextVariableException: Required context class hudson.FilePath is missing
```
The clear test report is at:
https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/ts_ofc/job/feature%252FtestJenkins/1/console